recipe/meta.yaml

```
@@ -7,12 +7,12 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e2ab66626fb57434ede7670ed8b2b2e8331919dbec78edb4d1d1646dfa0c73e7
+  sha256: 6c65e2122c8b62ee7985bb956369d05953c7704081f90b686a0fdfaf20933d2b
 
 build:
   number: 0
   #skip s390x as lack of nodejs and cartopy dependencies
-  skip: True  # [py<37 or s390x]
+  skip: True  # [py<310 or s390x]
 
```

```suggestion
requirements:
  host:
    - python
    - pip
    - setuptools
    - wheel
```
Try this. There can be a conda-build bug when the multi-output Python package is incorrectly handled

---

Also, add the skipping section to both outputs:
```
outputs:
  - name: {{ name }}-core
      build:
        skip: True  # [py<310 or s390x]
  - name: {{ name }}
      build:
        skip: True  # [py<310 or s390x] 
```
It prevents to pick up py38 by conda-build randomly (another conda-build bug?). See an example https://github.com/AnacondaRecipes/ray-packages-feedstock/blob/master/recipe/meta.yaml#L81C1-L82.

---

Another example https://github.com/AnacondaRecipes/modin-feedstock/blob/master/recipe/meta.yaml#L14-L19.

---

This worked!